beta/experience-platform-extension/offerdecisioning/offer-decisioning-extension.md

```diff
@@ -0,0 +1,209 @@
+# Offer Decisioning extension
+
+{% hint style="warning" %}
+The Adobe Experience Platform Offer Decisioning mobile extension is currently in BETA. Use of this extension is by invitation only. Please contact your Adobe Customer Success Manager to learn more and get access to the materials for this tutorial.
+{% endhint %}
+
+Adobe Experience Platform Offer Decisioning mobile extension can deliver personalized offers that are managed in Offer Decisioning. You can create your offers and other related objects using the Offer Decisioning user interface (UI) or APIs. To learn more about Offer Decisioning, please refer to [Offer Decisioning product documentation](https://experienceleague.adobe.com/docs/offer-decisioning/using/offer-decisioning-home.html?lang=en).
+
+## Prerequisites
+
+* IMS organization is enabled for edge decisioning
+* Offers, Activities created
+* Adobe Experience Platform Core and Edge mobile extensions have been implemented in the app
+* Launch UI config is published
+
+## Enable Offer Decisioning in Edge configuration
+
+To enable Offer Decisioning, you need to perform the following steps:
+
+1. In Adobe Experience Platform Launch, navigate to your mobile property and select Edge Configurations from the left panel, then select the configuration needs to be updated.
```

Comment:
Grammar: ...select the configuration **that** needs to be updated.

Edge Configurations is at the same level as Properties. Why do we need to navigate to the mobile property first? Isn't it the inlet that specifies the source, eg mobile app?

Reply:
Did you mean select the right IMS org first?